src/main/kotlin/blackjack/domain/card/Card.kt

```diff
@@ -0,0 +1,9 @@
+package blackjack.domain.card
+
+class Card(val suit: Suit, number: Number) {
```

Comment:
```suggestion
class Card(val suit: Suit, val number: Number) {
```
이렇게 정의해주고 4,5 라인은 없어도 될 것 같아요 : )